keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1717,45 +1618,76 @@ type ResourceClaimTemplateSpec struct {
 }
 ```
 
-##### Object references
+##### ResourceQuota
+
+At the moment, this type is only used to control whether admin access is
+allowed in a namespace. Other aspects may get added later.
```

Comment:
We should allow it be class (and make class required). That will cover most use cases, I think.

Reply:
Discussing quota details in https://github.com/kubernetes-sigs/wg-device-management/issues/24 so I will resolve this thread.